src/main/java/uk/gov/ons/ctp/response/action/service/ActionPlanJobService.java

```diff
@@ -60,88 +71,71 @@ public ActionPlanJob findActionPlanJob(final UUID actionPlanJobId) {
     return actionPlanJobRepo.findByActionPlanFK(actionPlan.getActionPlanPK());
   }
 
-  public List<ActionPlanJob> createAndExecuteAllActionPlanJobs() {
-    final List<ActionPlanJob> executedJobs = new ArrayList<>();
-    actionPlanRepo
-        .findAll()
-        .forEach(
-            actionPlan -> {
-              final Date lastExecutionTime =
-                  new Date(
-                      nowUTC().getTime() - appConfig.getPlanExecution().getDelayMilliSeconds());
-              if (actionPlan.getLastRunDateTime() == null
-                  || actionPlan.getLastRunDateTime().before(lastExecutionTime)) {
-                ActionPlanJob job =
-                    ActionPlanJob.builder()
-                        .actionPlanFK(actionPlan.getActionPlanPK())
-                        .createdBy(CREATED_BY_SYSTEM)
-                        .build();
-                job = createAndExecuteActionPlanJob(job);
-                if (job != null) {
-                  executedJobs.add(job);
-                }
-              } else {
-                log.debug(
-                    "Job for plan {} has been run since last wake up - skipping",
-                    actionPlan.getActionPlanPK());
-              }
-            });
-    return executedJobs;
+  public void createAndExecuteAllActionPlanJobs() {
+    List<ActionPlan> actionPlans = actionPlanRepo.findAll();
+    actionPlans.forEach(
+        actionPlan -> {
+          if (hasActionPlanBeenRunSinceLastSchedule(actionPlan)) {
+            log.with("actionPlanId", actionPlan.getId())
+                .with("actionPlanPK", actionPlan.getActionPlanPK())
+                .debug("Job for plan has been run since last wake up - skipping");
+          } else if (!hasActionableCases(actionPlan)) {
+            log.with("actionPlanId", actionPlan.getId())
```

Comment:
We have decided to go with snake case for all keys in logs to match the python services. `action_plan_id`

Reply:
Instead of going through and changing all the keys again, you could review https://github.com/ONSdigital/rm-action-service/pull/77 and get that merged in ;)